templates/portal/about.html

```diff
@@ -10,7 +10,8 @@
             <h3>About SciLifeLab Serve</h3>
             <p>SciLifeLab Serve (beta) is a platform for serving applications and machine learning models that is offered to life science researchers affiliated with a Swedish research institute and their collaborators. Currently, the service is running in beta testing mode. The service is available free of charge to researchers with and without an affiliation to SciLifeLab, to collaborators of these researchers, as well as to data-producing infrastructures in Sweden.</p>
             <h4>Infrastructure behind the service</h4>
-            <p>SciLifeLab Serve is developed and maintained by the <a href="https://scilifelab.se/data">SciLifeLab Data Centre</a>. The service was originally based on the code of open-source platform <a href="https://github.com/scaleoutsystems/stackn">STACKn</a>, developed by <a href="https://github.com/scaleoutsystems">Scaleout systems</a>, a spinoff from the Department of Information Technology at Uppsala University. The most up to date source code behind SciLifeLab Serve and other components that the service is built from are <a href="https://github.com/ScilifelabDataCentre/stackn/">available with an open source license on GitHub</a>.</p>
+            <p>SciLifeLab Serve is developed and maintained by the <a href="https://scilifelab.se/data">SciLifeLab Data Centre</a>. Initially the service was based on <a href="https://github.com/scaleoutsystems/stackn">STACKn</a>, a platform developed by <a href="https://github.com/scaleoutsystems">Scaleout systems</a> (a spinoff from the Department of Information Technology at Uppsala University). Since then, the SciLifeLab Data Centre team has expanded and improved the platform. The most up to date source code behind SciLifeLab Serve is <a href="https://github.com/ScilifelabDataCentre/stackn/">available with an open source license on GitHub</a>. We welcome any other organisations running a similar service using our code.</p>
```

Comment:
I think for many links similar or identical to this one we could try to do something like this:

<img width="642" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/8c1ef10b-e534-4694-9704-3e10514a52df">
<img width="616" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/e6a916f3-276e-4ca4-8650-95db5b2984be">
<img width="599" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/573b7e19-ed9f-4b7a-b9ff-917374523abf">

This is inspired by how substitutions work in restructured text
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions

Reply:
I didn't know about this as an option before so thank you for bringing it up! I'd say links directly in html like this are sufficient for us though, we don't use links excessively.